# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js
# Commit: da37e2d8a175
# Full Hash: da37e2d8a175fda08a98920d820dda0899b6aedf
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-13 09:36:48
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 15b92f98f62c -r da37e2d8a175 dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js
--- a/dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js	Thu Jun 12 18:16:52 2025 +0000
+++ b/dom/credentialmanagement/identity/tests/browser/browser_single_concurrent_identity_request.js	Thu Jun 12 18:16:53 2025 +0000
@@ -44,8 +44,8 @@
   ok(concurrentResponse.name, "expect a DOMException which must have a name.");
   is(
     concurrentResponse.name,
-    "InvalidStateError",
-    "Expected 'InvalidStateError', but got '" + concurrentResponse.name + "'"
+    "NotAllowedError",
+    "Expected 'NotAllowedError', but got '" + concurrentResponse.name + "'"
   );
 
   // Close tabs.